### DISCUSSION TITLE
Depedencytrack tool dashboard is showing server error 0

### DISCUSSION THREAD

**Discussion by biswajitdependencytrack** [community/community#47309] [GitHub Education]
I have installed docker and depedencytrack tool successfully. While opening the dependencytrack tool dashboard in http://localhost:8080, getting server error 0. Can anyone suggest how to resolve this issue. I did not get any OWASP portal to raise this ticket. If you know any OWASP link to raise this issue, please let me know. Please find attached the screenshot.
![4](https://user-images.githubusercontent.com/125332987/218633191-8c1df696-a59d-497d-be72-56f747496108.PNG)

**Comment by samus-aran** [comment]
Hi @biswajitdependencytrack,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [dependencytrack](https://docs.dependencytrack.org/). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
